app/src/components/common/button/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactNode, HTMLAttributes } from 'react'
+
+interface Props extends HTMLAttributes<HTMLButtonElement> {
```

Comment:
What's the difference between this component and a plain `button`?

Reply:
Is just a component reuse strategy. When we have the design we will apply styles to this component and reuse in different places of the application.
I think we will apply something like this:
```
const ButtonStyled = styled.button`
  align-items: center;
  background-color: ${props => props.theme.colors[props.type]};
  border-radius: ${props => props.theme.buttons.borderRadius};
  border: none;
  color: ${props => props.theme.buttons.color};
  cursor: pointer;
  display: flex;
  font-size: ${props => props.theme.buttons.fontSize};
  font-weight: ${props => props.theme.buttons.fontWeight};
  height: ${props => props.theme.buttons.height};
  justify-content: center;
`
```
Is just an example, I'm thinking aloud.